csi/csi.go

```diff
@@ -170,10 +166,11 @@ func (s *OsdCsiServer) getConn() (*grpc.ClientConn, error) {
 			return nil, fmt.Errorf("Failed to connect CSI to SDK uds %s: %v", s.sdkUds, err)
 		}
 	}
+	s.conn.GetState()
```

Comment:
what is this for?

Reply:
oops, that was leftover. I was checking that API. Will remove it. Thanks